### ISSUE TITLE
Join/leave/capture message

### ISSUE THREAD

**Issue by oy** [teeworlds/teeworlds#1569]
How about removing join/leave and capture messages from the chat and show them like kill messages up right corner using graphics?
+less translations
+more focus on real server chat messages

**Comment by Dune-jr** [comment]
Landil had made those icons back in the day:

![image](https://user-images.githubusercontent.com/355114/47604988-08aa4880-da01-11e8-9196-90c894f83fc0.png)

But the join/leave messages remained in the chat. It's kinda nice to still have IRC-style chat history, because you can see the history a bit better:

> **Pink tee joined**
> Pink tee: hey
> **Pink tee left**
> Dune: hey, Pink tee! would you like to do a DM 1v1?
> **Pink tee joined**
> Pink tee: sorry, what did I miss
> **Dune left**
> Red tee: huh, what a jerk

+1 for the graphics
but not sure about putting them on the right corner.

**Comment by nheir** [comment]
What about the capture time ?
It may be useful to keep at least this as message in the F1/log.

**Comment by ZombieToad** [comment]
i think alot of people wouldnt notice if their friends join the game then

**Comment by oy** [comment]
Yeah, could still drop a line (no translation) about it in the console so you have an overview.
The capture message should show the capture time. And these messages should be displayed longer than the kill messages so you notice it better.

**Comment by ChillerDragon** [comment]
No hate i am big Landil fanboi but these graphics are not teeish at all and they don't look good.
Probably the worst graphics Landis ever created.

**Comment by Dune-jr** [comment]
> No hate i am big Landil fanboi but these graphics are not teeish at all and they don't look good.
> Probably the worst graphics Landis ever created.

That's because that was very long ago, when he was young ;-)
I agree they should be redone, it was just to give an idea.

**Comment by sepulzera** [comment]
I would stick to popular/modern multiplayer games here. Dota2 and CS:GO display those messages in the chat, too. I do not see any reason to deviate from it.

> But the join/leave messages remained in the chat. It's kinda nice to still have IRC-style chat history, because you can see the history a bit better - *Dune-jr*

+1

> Yeah, could still drop a line (no translation) about it in the console so you have an overview. - *oy*

Console is for "techies", not the common player.
